### ISSUE TITLE
ilert_connector allows to setup unsupported type "microsoft_teams_chat"

### ISSUE THREAD

**Issue by alexs77** [iLert/terraform-provider-ilert#7]
```hcl
resource "ilert_connector" "sharedien_notifications_chat" {
  name = "DevOps Notifications Chat"
  type = "microsoft_teams_chat"
}
```

In the web interface, this is then shown like this:

![iLert Web interface ⇒ Connectors](https://user-images.githubusercontent.com/781361/156123801-37c4f3e8-98aa-4af7-8d9a-baceb2b407c1.png)

- Type: `???extension_microsoft_teams_chat???`
- Icon URL: https://sharedien.ilert.com/extension/RES_NOT_FOUND

This doesn't seem to be right.

```
askwar@a00c053 ~/C/ilert-test> terraform version
Terraform v1.1.6
on linux_amd64
+ provider registry.terraform.io/ilert/ilert v1.4.7
```

**Comment by krystianity** [comment]
Hi @alexs77 thank you for reporting this. This should not be possible.

**Comment by yacut** [comment]
fixed in v1.5.1
